src/prasadSDec2019/Assignment3/Question10.java

```diff
@@ -0,0 +1,60 @@
+package prasadSDec2019.Assignment3;
+
+//WAP to find Factorial series for user defined number 
+//a. Input number :- 5 
+//b. Output :- 120 
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Question10 {
+	
+	void sumOfArray(int arr1[],int arr2[]) {
+		int sum;
```

Comment:
instead of creating two for loop in one method create one separate method which accept one int type array 
and return it sum.